providers/openfeature-provider-flipt-client/src/openfeature/contrib/provider/flipt-client/__init__.py

```diff
@@ -0,0 +1,235 @@
+from typing import Optional, Union, Any
+
+from enum import Enum
+from openfeature.provider import AbstractProvider, Metadata
+from flipt_client.models import (
+    FetchMode, 
+    AuthenticationStrategy, 
+    ClientTokenAuthentication,
+    JWTAuthentication,
+    ClientOptions
+)
+from flipt_client import FliptEvaluationClient
+
+from openfeature.evaluation_context import EvaluationContext
+from openfeature.exception import ErrorCode
+from openfeature.flag_evaluation import FlagResolutionDetails, Reason, FlagType
+
+# Expose the provider and relevant models
+__all__ = ['FliptClientProvider', 'ClientTokenAuthentication', 'JWTAuthentication', 'FetchMode'] 
+
+
+class FliptClientProvider(AbstractProvider):
+    '''Wrapper for the Flipt Evaluation Client with the standard OpenFeature Provider interface'''
+
+    def __init__(
+        self,
+        base_url: str,
+        namespace: str = 'default',
+        *,
+        update_interval: Optional[int] = None,
+        authentication: Optional[AuthenticationStrategy] = None,
+        reference: Optional[str] = None,
+        fetch_mode: Optional[FetchMode] = None
+    ):
+        self._client=FliptEvaluationClient(
+            namespace=namespace,
+            opts=ClientOptions(
+                url=base_url,
+                update_interval=update_interval,
+                authentication=authentication,
+                reference=reference,
+                fetch_mode=fetch_mode
+            )
+        )
+    
+    def get_metadata(self) -> Metadata:
+        return Metadata(
+            name='OpenFeature Flipt Client-Side Provider',
```

Comment:
```suggestion
            name='Flipt',
```

I'd recommend keeping this as short as possible since it may be used in telemetry and logs.